proseco/catalog.py

```diff
@@ -481,3 +514,50 @@ def filt(tbl, id_set):
     aca.add_column(idx, index=1)
 
     return aca
+
+ROLL_TABLE = Table.read("""
+pitch   rolldev
+46      0
```

Comment:
The last table I have from the FOT is included in a thttps://github.com/sot/aca_lts_eval/pull/52/commits/e5bba740c9a91cdba3be2f040d8c32b31122ac6c